_docs/operations/overcommitment.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: guide
+title: StorageOS Docs - Overcommitment
+anchor: operations
+module: operations/overcommitment
+---
+
+# Overcommitment
+
+Overcommitment is a way of creating volumes whose size exceeds the actual
```

Comment:
All volumes in StorageOS are thinly provisioned and support inline compression.Therefore volumes only occupy capacity in the pool based on the data actually written to the StorageOS volume. StorageOS allows pools to be overcommitted. This allows the creation of volumes whose size exceeds the actual storage presented by underlying nodes. It can be useful if you anticipate writing a lot of data to a volume eventually but you do not wish to pay for storage that is not currently
 being used.

Your paragraph seemed to be missing some words between "inline compression" and "only". So I went with the above.